lms/static/sass/views/_shoppingcart.scss

```diff
@@ -637,390 +532,480 @@
-          display: inline-block;
-          float: right;
-          line-height: 20px;
-          color: $dark-gray1;
+        // STATE: hover
+        &:hover {
+          box-shadow: none;
+          background: $m-blue-d2;
         }
       }
+
+      p {
+        @include text-align(right);
+        padding: ($baseline*0.65) 0;
+        font-family: $sans-serif;
+      }
+
+      form {
+        position: relative;
+      }
+
+      .fa-caret-right {
+        @include right($baseline*1.5);
+        @extend %t-icon3;
+        position: absolute;
+        top: ($baseline*1.1);
+        color: $white;
+      }
+
+      label.pull-right {
+        @include float(right);
+        @include text-align(right);
+        @include line-height(13.51);
+        display: inline-block;
+        padding: ($baseline/2) ($baseline*1.25);
+        color: $dark-gray1;
+        font-family: $sans-serif;
+        font-style: normal;
+      }
     }
-  .disclaimer{
-    color: $light-gray2;
+  }
+
+  .disclaimer {
+    @include text-align(right);
+    @extend %t-light;
     padding: ($baseline/2) 0;
-    text-align: right;
-    font-weight: 300;
+    color: $light-gray2;
   }
-  h3{
-    font-family: "Open Sans",Verdana,Geneva,sans-serif,sans-serif;
-    font-size: 16px;
-    font-weight: 400;
-    padding: 30px 20px;
+
+  h3 {
+    @extend %t-icon5;
+    @extend %t-regular;
+    padding: ($baseline*1.5) $baseline;
     color: $dark-gray1;
+    font-family: $sans-serif;
   }
-  .billing-data{
+
+  .billing-data {
     display: table;
     width: 100%;
-    h3{
-      padding: 12px 0px;
-      color: $dark-gray1;
-      font-size: 17px;
+
+    h3 {
+      @include font-size(17);
       margin-bottom: ($baseline/4);
+      padding: ($baseline*0.6) 0;
+      color: $dark-gray1;
     }
-    .row{
+
+    .row {
       display: table-row;
     }
-    .col-half{
-      width: 45%;
-      float: left;
-      background: $light-gray1;
-      padding: 20px;
-      border-radius: 4px;
-      margin-bottom: ($baseline*0.75);
-      min-height: 240px;
-      &:nth-child(even){
-        margin-left: 30px;
-      }
-      .data-group{
+
+    .col-half {
+      @include float(left);
+      @include border-radius(4px);
       margin-bottom: ($baseline*0.75);
-        label{
-        display: block;
-        font-family: "Open Sans",Verdana,Geneva,sans-serif,sans-serif;
-        font-size: 16px;
-        font-style: normal;
-        font-weight: 400;
-        color: $dark-gray2;
+      background: $light-gray1;
+      padding: $baseline;
+      width: 45%;
+      min-height: ($baseline*12);
+
+      // CASE: index of child is even
+      &:nth-child(even) {
+        @include margin-right($baseline*1.5);
       }
-      input{width: 100%;margin-bottom: ($baseline/4);}
-      &:nth-child(4n){
-        margin-right: 0;
+
+      .data-group {
+        margin-bottom: ($baseline*0.75);
+
+        label {
+          @extend %t-icon5;
+          @extend %t-regular;
```

Comment:
using the other typography extends will remove the need to add the font-family rule below.